### DISCUSSION TITLE
Converting from Plain Text or md to Supernote does not properly render line breaks and paragraph breaks

### DISCUSSION THREAD

**Discussion by affvalente** [standardnotes/forum#2940] [General]
Converting from any type of note to supernote does not respect empty lines

## Title
A

B

C

gets converted to

## Title
A
B
C

in Supernote

**Comment by effieeee** [comment]
Hmm, I've only been able to reproduce this when coming from plain text or markdown notes so far. When you insert line breaks and empty spaces on a rich text note, does it not get converted properly in the Super note's preview window?

**Comment by affvalente** [comment]
Thanks - my only use case is from plain text and md to super note. I don’t use rich text

**Comment by sonjoonho** [comment]
I think this is a styling issue, the paragraph break conversion works fine.

As is: 

![image](https://user-images.githubusercontent.com/20669173/216207626-f617b3c6-e54c-43b5-8e67-65615bb92e02.png)

Adding some margin to each paragraph block makes this readable, and should be the default IMO (should I file a feature request?):

![image](https://user-images.githubusercontent.com/20669173/216207858-7e650421-b36e-4ca3-b023-7b3367f7c243.png)
